app/models/vm/operations.rb

```diff
@@ -22,7 +22,7 @@ def cockpit_url
   end
 
   def ipv4_address
-    ipaddresses.find { |ip| (IPAddr.new ip).ipv4? }
+    ipaddresses.find { |ip| IPAddr.new(ip).ipv4? }
```

Comment:
Just wanted to say that this is probably how this one should have been written in the first place.